cadence/contracts/DapperUtilityCoin.cdc

```diff
@@ -57,78 +49,85 @@ pub contract DapperUtilityCoin: FungibleToken {
         // created Vault to the context that called so it can be deposited
         // elsewhere.
         //
-        pub fun withdraw(amount: UFix64): @FungibleToken.Vault {
+        access(FungibleToken.Withdrawable) fun withdraw(amount: UFix64): @DapperUtilityCoin.Vault {
```

Comment:
Looks like you're using an old version of the token standards here. The entitlement should be `FungibleToken.Withdraw`, not `Withdrawable`